locations/spiders/Amazons.py

```diff
@@ -0,0 +1,143 @@
+import scrapy
+from scrapy.linkextractors import LinkExtractor
+from scrapy.spiders import CrawlSpider, Rule
+import  requests
+import random
+from locations.items import  GeojsonPointItem
+
+User_Agents = ["Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/44.0.2403.157 Safari/537.36",
```

Comment:
I'm not very excited about this. Did the spider fail completely without multiple user agents?

Reply:
Actually due to lot of requests while testing the spider kept failing, so user agents moreover helps the spider